tests/20-verify.js

```diff
@@ -0,0 +1,150 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import {bs58Decode, bs58Encode} from './helpers.js';
+import {verificationFail, verificationSuccess} from './assertions.js';
+import {endpoints} from 'vc-api-test-suite-implementations';
+import {generateTestData} from './vc-generator/index.js';
+
+// only use implementations with `eddsa-2022` verifiers.
+const {match, nonMatch} = endpoints.filterByTag({
+  tags: ['eddsa-2022'],
+  property: 'verifiers'
+});
+
+describe('eddsa-2022 (verify)', function() {
+  let credentials;
+  before(async function() {
+    credentials = await generateTestData();
+  });
+  describe('Data Integrity (verifier)', function() {
+    // this will tell the report
+    // to make an interop matrix with this suite
+    this.matrix = true;
+    this.report = true;
+    this.rowLabel = 'Test Name';
+    this.columnLabel = 'Verifier';
+    this.implemented = [...match.keys()];
+    this.notImplemented = [...nonMatch.keys()];
+    for(const [columnId, {endpoints}] of match) {
+      describe(columnId, function() {
+      // wrap the testApi config in an Implementation class
+        const [verifier] = endpoints;
+        it('If the "proof" field is missing or invalid, a MALFORMED error ' +
+          'MUST be returned.', async function() {
+          this.test.cell = {columnId, rowId: this.test.title};
+          const credential = credentials.clone('issuedVc');
+          delete credential.proof;
+          await verificationFail({credential, verifier});
+        });
+        it('If the "type" field is missing or invalid, a MALFORMED error ' +
+          'MUST be returned.', async function() {
```

Comment:
This test can't be testing if the field is missing OR invalid. It is testing that the field is missing, full stop. If not, it is testing that the field exists and is invalid, full stop. Please revise the test description to reflect what is being tested.

Reply:
addressed here: https://github.com/w3c-ccg/di-eddsa-2022-test-suite/pull/4/commits/836e6dadc05bb5d5ebfc17562c6e6364e462b7a6